packages/fakta/omsorg-og-rett/src/OmsorgOgRettFaktaIndex.spec.tsx

```diff
@@ -19,6 +22,7 @@ describe('OmsorgOgRettFaktaIndex', () => {
 
     const utils = render(<HarAksjonspunktForAvklarAleneomsorg submitCallback={lagreVurdering} />);
 
+    expect(screen.queryByText('Rettighetstype')).toBeNull();
```

Comment:
Skal du sjekka om teksten ikkje er i dokument så bruk heller .not.toBeInTheDocument()